node-runner-cli/templates/radix-fullnode-compose.yml.j2

```diff
@@ -109,7 +108,6 @@ services:
     depends_on:
       - database_migrations
     image: {{gateway.data_aggregator.repo}}:{{gateway.data_aggregator.release}}
-    platform: linux/amd64
```

Comment:
Not sure why we had this ?  Now we have arm images, hence we removing this line?

Reply:
Yes exactly. I think I added it back in February because I was running the resulting docker-compose locally and there was no arm images available, resulting in "cant find the image"-error.